roda-ui/roda-wui/src/main/resources/config/i18n/client/ClientMessages.properties

```diff
@@ -284,6 +284,7 @@ backButton:Back
 cancelButton:Cancel
 revertButton:Revert
 removeButton:Remove
+unsubscribeButton:Unsubscribe

```

Comment:
move this to the section that has the messages related to the local instance and add `localInstance` prefix

Reply:
Done